app/assets/javascripts/application/rvt_helper.js

```diff
@@ -0,0 +1,46 @@
+function removeRvtHelper(userId) {
+    const url = `/admin/rvt_helpers/${userId}/remove`
+    fetch(url, {
+        method: 'post',
+        headers: {'X-CSRF-Token': getCsrfToken()}, // this line is important, if this content-type is not set it wont work
+    }).then(function (response) {
+        return response.text();
```

Comment:
We're not really using the response here, so I think this can be omitted. (also at `addRvtHelper`)